src/bilder/images/edxapp/templates/common_values.yml

```diff
@@ -219,12 +218,12 @@ COURSE_CATALOG_VISIBILITY_PERMISSION: see_exists
 CREDENTIALS_INTERNAL_SERVICE_URL: http://localhost:8005
 CREDENTIALS_PUBLIC_SERVICE_URL: http://localhost:8005
 CREDIT_PROVIDER_SECRET_KEYS: {}
-CROSS_DOMAIN_CSRF_COOKIE_DOMAIN: .mitxonline.mit.edu  # MODIFIED
-CROSS_DOMAIN_CSRF_COOKIE_NAME: edxapp-mitxonline-qa-csrftoken  # MODIFIED
+CROSS_DOMAIN_CSRF_COOKIE_DOMAIN: {{ key "edxapp/cookie-domain" }}  # MODIFIED
+CROSS_DOMAIN_CSRF_COOKIE_NAME: edxapp-{{ env "ENVIRONMENT" }}-csrftoken  # MODIFIED
 CSRF_COOKIE_SECURE: true  # MODIFIED
 CSRF_TRUSTED_ORIGINS:  # MODIFIED
   - learning-mfe-qa.mitxonline.mit.edu
```

Comment:
Should this have a key too?

Reply:
I'm punting on that for the time being, but yes, it will need to have a key as well.